sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/OperationScreenBuilder.kt

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright 2020 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package br.com.zup.beagle.sample.builder
+
+import br.com.zup.beagle.ext.setStyle
+import br.com.zup.beagle.operations.builtin.*
+import br.com.zup.beagle.widget.context.Bind
+import br.com.zup.beagle.widget.context.ContextData
+import br.com.zup.beagle.widget.context.constant
+import br.com.zup.beagle.widget.context.expressionOf
+import br.com.zup.beagle.widget.core.EdgeValue
+import br.com.zup.beagle.widget.core.UnitValue
+import br.com.zup.beagle.widget.layout.*
```

Comment:
<table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-backend-kotlin/blob/6a35e23efdaf4bacd5c464386e37eba1c8d8425c/sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/OperationScreenBuilder.kt#L27"></span>br.com.zup.beagle.widget.layout.* is a wildcard import. Replace it with fully qualified imports.</td>
    </tr>
  </tbody>
</table>